pkg/tenant/auth.go

```diff
@@ -218,7 +218,7 @@ func (s *Server) Login(c *gin.Context) {
 		// Get member from the database by their email.
 		var member *db.Member
 		if member, err = db.GetMemberByEmail(c, orgID, params.Email); err != nil {
-			sentry.Error(c).Err(err).Msg("could not get member from the database")
+			sentry.Error(c).Str("email", params.Email).Str("orgID", orgID.String()).Err(err).Msg("could not get member from the database")
```

Comment:
Should the sentry error in `Register` be updated to display the string details as well when this method is used?

Reply:
Sure why not